src/Command/GuideCommand.php

```diff
@@ -28,7 +28,7 @@ final class GuideCommand extends Command
     use CommandTrait;
 
     // Regular expression to match comment
-    private const REGEX = '/^\s*\/\/\s/';
+    public const REGEX = '/^\s*\/\/\s/';
```

Comment:
why?

Reply:
To reuse it in TestGuideCommand when checking for frontmatter